eu_dgc_v1_schema.json

```diff
@@ -2,12 +2,33 @@
     "$schema": "http://json-schema.org/draft/2020-12/schema#",
     "$id": "https://github.com/ehn-digital-green-development/hcert-schema/eu_dgc_v1",
     "title": "Digital Green Certificate",
-    "description": "Proof of vaccination, test results or recovery according to EU eHN, version 1.0, including certificate metadata; According to 1) REGULATION OF THE EUROPEAN PARLIAMENT AND OF THE COUNCIL on a framework for the issuance, verification and acceptance of interoperable certificates on vaccination, testing and recovery to facilitate free movement during the COVID-19 pandemic (Digital Green Certificate) - https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0130 2) Document \"Value Sets for the digital green certificate as stated in the Annex ...\", abbr. \"VS-2021-04-08\" - https://webgate.ec.europa.eu/fpfis/wikis/x/05PuKg",
+    "description": "Proof of vaccination, test results or recovery according to EU eHN, version 1.0, including certificate metadata; According to 1) REGULATION OF THE EUROPEAN PARLIAMENT AND OF THE COUNCIL on a framework for the issuance, verification and acceptance of interoperable certificates on vaccination, testing and recovery to facilitate free movement during the COVID-19 pandemic (Digital Green Certificate) - https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0130 2) Document \"Value Sets for the digital green certificate as stated in the Annex ...\", abbr. \"VS-2021-04-14\" - https://webgate.ec.europa.eu/fpfis/wikis/x/05PuKg",
     "type": "object",
     "required": [
+        "v",
+        "id",
+        "is",
         "sub"
     ],
     "properties": {
+        "v": {
+            "title": "Schema version",
+            "description": "Version of the schema",
+            "type": "string",
+            "example": "v1.0"
+        },
+        "id": {
+            "title": "Identifier",
+            "description": "Identifier of the DGC (initially called UVID (V for vaccination), later renamed to DGCI)",
+            "type": "string",
+            "example": "01AT42196560275230427402470256520250042"
+        },
+        "is": {
```

Comment:
Do we really want this to be required? The CWT holds the issuing country and that will probably be sufficient for most verifiers.

Also. The signer certificate (i.e., the signer of the CWT) will indicate in its subject DN the same information.